free-1.md

```diff
@@ -56,13 +64,16 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
+I think it would be a little more system aspects because I would have to figure out what words mean what and in what contexts while the english is pretty standardized already.
```

Comment:
As we read in our second hw readings, natural language isn't truly natural language in a DSL, so deciding on how exact language would be processed, so that the user would intuitively know what they were doing, would require a lot of language design.